keyboards/handwired/swiftrax/cowfish/config.h

```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0x7161
-#define PRODUCT_ID      0x5239
+#define VENDOR_ID       0x04D8
```

Comment:
This VID belongs to Microchip.

Reply:
Thanks reverted it. The PID was given to me by them

Reply:
if they've granted you a PID, then aren't you allowed to use their VID?

Reply:
![image](https://user-images.githubusercontent.com/48102030/89705041-a0470580-d90e-11ea-892e-772e5cc2aa89.png)

Reply:
Perfect, cheers for that.